app/services/billing_entities/update_service.rb

```diff
@@ -74,6 +75,7 @@ def call
         end
 
         handle_invoice_custom_sections if params.key?(:invoice_custom_section_ids) || params.key?(:invoice_custom_section_codes)
+        handle_einvoincing_setting if params.key?(:country) && billing_entity.einvoicing
```

Comment:
I'm not a big how this is handled. First, we assign something and 50+ lines later we override the value. I'd prefer if the correct value was set in the assignment.

**More importantly,** since we have validation for this, shouldn't we let the service handle the validation via the ActiveRecord::InvalidRecord exception instead of assigning another value silently? Like the create service?